### ISSUE TITLE
Import fails once content of the article is included

### ISSUE THREAD

**Issue by herr-schaft** [omnivore-app/obsidian-omnivore#193]
Importing articles from Omnivore to Obsidian fails, once I try to include the content of the article.
Tried a couple of article templates. All seem to work fine – importing labels, highlights, notes etc. – until I try and add the content of the artice. Neither {{{ content }}} nor {{{content}}} without spacing works.

The plugin will start 'fetching articles' but then seems to give up. No articles are fetched nor is an error displayed. The Obsidian log shows the standard:
```
2024-02-10 23:09 - Uploading file .obsidian/plugins/obsidian-omnivore/data.json
2024-02-10 23:09 - Upload complete .obsidian/plugins/obsidian-omnivore/data.json
2024-02-10 23:09 - Fully synced
``` 

This would be my complete article template – which works fine once stripped of the content:
```
#omnivore {{#labels}} #{{name}} {{/labels}}

{{#highlights.length}}
## Highlights

{{#highlights}}
> {{{text}}} [⤴️]({{{highlightUrl}}}) {{#labels}} #{{name}} {{/labels}} ^{{{highlightID}}}
{{#note}}

{{{note}}}
{{/note}}

{{/highlights}}
{{/highlights.length}}

{{{ content }}}

[Read on Omnivore]({{{omnivoreUrl}}})
[Read Original]({{{originalUrl}}})
``` 

The issue seems related to #185 and #179, which do not explicitly mention the article content, though.

**Comment by MaryseBucking** [comment]
@herr-schaft: thanks, this is exactly the problem why it isn't syncing properly for me. When deleting the content from my template, it syncs everything. Not what I want however.

**Comment by bayang** [comment]
@MaryseBucking  are you using a custom query to sync ? 
I was having the same problem as you : no more articles being imported in obsidian and the logs showing no errors at all. Just nothing was imported.

This behaviour started a few weeks ago , and the last answer in this issue fixed it for me : https://github.com/omnivore-app/omnivore/issues/3499  

-> tl;dr; -> `Hi, if you have a custom query you should add in:all to get archived items so something like in:all label:Obsidian`

**Comment by herr-schaft** [comment]
@bayang, the query in:archive works fine for me (until I try to add the content of an article).  Meaning: in:all isn't the only query working.

**Comment by bayang** [comment]
@herr-schaft  There are different issues I think. But on my side, a custom query without the `in` part is not working anymore.

**Comment by sywhb** [comment]
hey @bayang @herr-schaft , i just released a new version v1.9.4 to address the custom query issue.

could you please try it out?

**Comment by bayang** [comment]
@sywhb here are the logs and my observations after upgrading :  

```
09:15:54.355 plugin:obsidian-omnivore:14254 obsidian-omnivore: advanced filter is replaced with all filter
09:15:54.355 plugin:obsidian-omnivore:14257 obsidian-omnivore: custom query is set to in:all (in:all label:omnivore)
```

I had set my query to `in:all label:omnivore` and it seems your update sets it to `in:all (in:all label:omnivore)`

If I understand correctly to work the query has to include the `in:` query ? 

Maybe just put it in bold in the docs and be done with it so that people know it.

I tried to import with just `label:omnivore` and it didn't work, I set it back to `in:all label:omnivore` and it worked
.

**Comment by herr-schaft** [comment]
@sywhb sadly no change for me. Synching items continues to work fine until I add the `{{{ content }}} `of an article to the article template. This is regardless of the custom query being `in:all (in:archive)` or just `in:archive`.

**Comment by sywhb** [comment]
> @sywhb here are the logs and my observations after upgrading :
> 
> ```
> 09:15:54.355 plugin:obsidian-omnivore:14254 obsidian-omnivore: advanced filter is replaced with all filter
> 09:15:54.355 plugin:obsidian-omnivore:14257 obsidian-omnivore: custom query is set to in:all (in:all label:omnivore)
> ```
> 
> I had set my query to `in:all label:omnivore` and it seems your update sets it to `in:all (in:all label:omnivore)`
> 
> If I understand correctly to work the query has to include the `in:` query ?
> 
> Maybe just put it in bold in the docs and be done with it so that people know it.
> 
> I tried to import with just `label:omnivore` and it didn't work, I set it back to `in:all label:omnivore` and it worked .

thanks for the feedback!

if `in` filter were not set in the custom query, it would use `in:inbox` by default

**Comment by sywhb** [comment]
> @sywhb sadly no change for me. Synching items continues to work fine until I add the `{{{ content }}} `of an article to the article template. This is regardless of the custom query being `in:all (in:archive)` or just `in:archive`.

that's kind of weird because it would show an error message in the console logs if it failed to sync the content.

i suspect it is still syncing.
would you mind opening you `.obsidian/plugins/obsidian-omnivore/data.json` when it starts to sync data from omnivore?
there is a key `"syncing"` in the json file and if it finished syncing, the value would be `false`.

**Comment by PatNei** [comment]
I get the same error, the console reveals the following error:
<img width="778" alt="image" src="https://github.com/omnivore-app/obsidian-omnivore/assets/49490990/1a43d3dc-cfe7-41cc-9614-5a7c9f24d3f9">

**Comment by MaryseBucking** [comment]
I updated to the latest version, and it kept saying it was syncing, but nothing happened. I noticed that my 'last sync' was emptied in the settings, so I manually set it to the first of this month. Then tried again. After a while it gave an error and when I tried to sync again by manually starting the sync, it immediately said: failed to fetch articles. I've added the log. 
[obsidian.md-1708763381316.log](https://github.com/omnivore-app/obsidian-omnivore/files/14392183/obsidian.md-1708763381316.log)
